domains/halil.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "portfolio",
+  "repo": "https://github.com/halil-dll/halil-dll.github.io",
+  "owner": {
+    "username": "halil-dll",
+    "email": "halil@pissmail.com"
+  },
+  "record": {
+    "CNAME": "halil-dll.github.io"
```

Comment:
Yo should fix your website, everything is a 404 except home page that has broken css and just displays an home